### DISCUSSION TITLE
Thank you for your contribution

### DISCUSSION THREAD

**Discussion by charlye30** [o0Zz/sys-con#26] [General]
Hi, I'm just writting to appreciate your work. I'm testing my old Thrustmaster Dual Analog 4. Works and I'm testing for bugs. 
Too much time waiting for this and don't knowing why are not working in the old sys-con. Thank u

**Comment by o0Zz** [comment]
Happy to see it work on your side. If you created a mapping for your controller, you could share it. Regarding why it don't works with the old sys-con : The reason is simple, the previous developer only handled xbox controllers and psx controllers while this version also support PC controller. All these controllers works and behave differently with different protocol. This fork rely on the HID standard which means that all controller compatible with PC should be compatible with the switch. Thanks
